README.md

```diff
@@ -4,7 +4,7 @@
 
 <p align="center"><img src=https://raw.githubusercontent.com/cuappdev/assets/master/eatery/Eatery-Long-Logo.png width=500 /></p>
 
-Eatery was the first app made by [AppDev](http://cornellappdev.com/), an engineering project team at Cornell University focused on mobile app development. Eatery provides an easy and accessible way to browse the hours/menus of the dining locations on campus as well as keep track of your dining history. Download the current release on the [Apple App Store](https://itunes.apple.com/us/app/id1089672962)
+Eatery was the first app made by [AppDev](http://cornellappdev.com/), an engineering project team at Cornell University focused on mobile app development. Eatery provides an easy and accessible way to browse the hours/menus of the dining locations on campus as well as keep track of your dining history. Download the current release on the [Apple App Store](https://itunes.apple.com/us/app/id1089672962).
```

Comment:
I'm not sure this should be this way. Maybe we could pull this out into another sentence or maybe change the spacing a bit. Also, I stylistically disagree with periods at the end of paragraphs because they add too much clutter, but to each their own. Looks good to me